bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -2772,6 +2772,12 @@
 			return escapedText;
 		};
 
+		_t.unEscapeHtml = function(html) {
+			var text = document.createElement("textarea");
+			text.innerHTML = html;
+			return text.value;
```

Comment:
Is it a clean way to do ?
Doesn't this new element appear in the page after creation ?

Reply:
No, it doesn’t. This is the most commended way I could find.

Reply:
In the linked URL, someone proposed to remove the created element. So maybe something like that is even better ?
```
var text = document.createElement("textarea"),
    decodedHtml;;
text.innerHTML = html;
decodedHtml = text.value;
text.remove();
return decodedHtml;
```

Reply:
I also read that it is garbage collected anyway as no reference is held to it after returning from the function, and that remove is not available in all versions of js. The last point is probably not an issue anymore with versions currently used in browsers, but I don’t see why risking it.